docs/quickstart.ipynb

```diff
@@ -2580,11 +2779,185 @@
    "source": [
     "For more details see the vignette on [Difference-in-Differences Estimation](https://py-econometrics.github.io/pyfixest/difference-in-differences.html)."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Weighted Lease Squares (WLS)\n",
```

Comment:
>     This requires that the weights that are passed in are proportional to the inverse of the error variance.   		

This is technically not true and something that is very confusing in the statsmodels docs. The statsmodels docs discuss Feasible Weighted Least Squares for the purpose of correcting for heteroskedasticity (because the relation between dependent variable and covariates is quadratic, but we fit a linear model, we have homoskedasticity by definition). 

    This requires that the weights that are passed in are proportional to the inverse of the error variance.   		

This is technically not true and something that is very confusing in the statsmodels docs. The statsmodels docs discuss Feasible Weighted Least Squares for the purpose of correcting for heteroskedasticity. 


But one can simply fit a model with weights being applied to different observations. I.e. from wikipedia, we can simply minimize the following least squares objective function 

![image](https://github.com/py-econometrics/pyfixest/assets/19531450/e5b3877e-3adc-4d4b-8bde-afd4ba99c50b)

Optimally, I would like us to start with a very brief section that practically says "you can apply weights, here is how you do it using either frequency weights or precision weights". 

Here we could show an example such as this: 

```{python}
import pyfixest as pf 

data = pf.get_data(model = "Fepois")[["X1","Y", "f1"]]
data.head()
# data of size N = 1000
print(data.shape)
data_agg = data.groupby(["X1","Y", "f1"]).size().reset_index().rename(columns = {0: "count"})
data_agg.head()
# data of size N << 1000: useful for reducing memory 
print(data_agg.shape)

fit_ols = pf.feols("Y ~ X1 | f1", data = data, vcov = "iid")
fit_wls = pf.feols("Y ~ X1 | f1", data = data_agg, vcov = "iid", weights = "count", weights_type = "fweights")
pf.etable([fit_ols, fit_wls])
# identical results and standard errors
```

[And also link to the "Causal Inference for the Brave and True" chapter on WLS. ](https://matheusfacure.github.io/python-causality-handbook/06-Grouped-and-Dummy-Regression.html#grouped-and-dummy-regression)

And then we can go on to motivate Feasible WLS a little bit better. In this case, I would like to add a sentence below the plot that explains that the conditional Variance $Var(Y|X)$ is increasing / changing in X - in other words, we have heteroskedasticity. And to tackle heteroskedasticity, we can either use robust standard errors or use our knowledge of the data generating process by weighting our regression model appropriately. And then we compare the WLS standard errors with the HC robust standard errors.